content/en/docs/Dashboard/_index.md

```diff
@@ -21,6 +21,8 @@ in their creation, execution, and completion
 
 ## Installation
 
+This installation is available through [katacoda](https://katacoda.com/ncskier/scenarios/tekton-dashboard).
```

Comment:
Maybe 'A tutorial is available...' ?
Also 'Katacoda' (capital 'K')

I think it might be a good idea to put this in a separate section so it's clear it's not required for the actual installation steps. A `## Tutorial` section just above might be a good place for it. What do you think?